specs/_features/epbs/builder.md

```diff
@@ -0,0 +1,116 @@
+# ePBS -- Honest Builder
+
+This is an accompanying document which describes the expected actions of a "builder" participating in the Ethereum proof-of-stake protocol.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Builders attributions](#builders-attributions)
+  - [Constructing the payload bid](#constructing-the-payload-bid)
+  - [Constructing the `BlobSidecar`s](#constructing-the-blobsidecars)
+  - [Constructing the execution payload envelope](#constructing-the-execution-payload-envelope)
+  - [Honest payload withheld messages](#honest-payload-withheld-messages)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+With the ePBS Fork, the protocol includes new staked participants of the protocol called *Builders*. While Builders are a subset of the validator set, they have extra attributions that are optional. Validators may opt to not be builders and as such we collect the set of guidelines for those validators that want to act as builders in this document. 
+
+## Builders attributions
+
+Builders can submit bids to produce execution payloads. They can broadcast these bids in the form of `SignedExecutionPayloadHeader` objects, these objects encode a commitment to reveal an execution payload in exchange for a payment. When their bids are chosen by the corresponding proposer, builders are expected to broadcast an accompanying `SignedExecutionPayloadEnvelope` object honoring the commitment. 
+
+Thus, builders tasks are divided in two, submitting bids, and submitting payloads. 
+
+### Constructing the payload bid
+
+Builders can broadcast a payload bid for the current or the next slot's proposer to include. They produce a `SignedExecutionPayloadHeader` as follows. 
+
+Prior to constructing a payload, the builder **MUST** have a full `InclusionList` object from the proposer matching `state.previous_inclusion_list_proposer`. The signed summary for this inclusion list will be needed when revealing the full execution payload (see below). 
```

Comment:
may be totally remove inclusion list for now?

Reply:
it will be removed in the EIP, I'll open a new PR in this repo based on Electra soon